client/my-sites/plans/current-plan/index.jsx

```diff
@@ -131,7 +123,11 @@ class CurrentPlan extends Component {
 				<QuerySitePlans siteId={ selectedSiteId } />
 				{ shouldQuerySiteDomains && <QuerySiteDomains siteId={ selectedSiteId } /> }
 
-				{ ! showThankYou && <PlansNavigation path={ path } /> }
+				<Dialog isVisible={ showThankYou }>
+					{ isFreePlan ? <FreePlanThankYouCard /> : <PaidPlanThankYouCard /> }{' '}
```

Comment:
Why the `{' '}`?

Reply:
That was unintended. Removed in 6c600afffe7b7e9fa5b27c5b69140df25658772f. Thanks for spotting!